apps/vessel-history/features/profile/Profile.tsx

```diff
@@ -90,54 +144,7 @@ const Profile: React.FC = (props): React.ReactElement => {
         })
       )
       if (fetchVesselByIdThunk.fulfilled.match(action as any)) {
-        const vesselDataset = datasets
-          .filter((ds) => ds.id === dataset)
-          .slice(0, 1)
-          .shift()
-
-        if (vesselDataset) {
-          const trackDatasetId = getRelatedDatasetByType(vesselDataset, DatasetTypes.Tracks)?.id
-          if (trackDatasetId) {
-            const eventsRelatedDatasets = getRelatedDatasetsByType(
-              vesselDataset,
-              DatasetTypes.Events
-            )
-
-            const eventsDatasetsId =
-              eventsRelatedDatasets && eventsRelatedDatasets?.length
-                ? eventsRelatedDatasets.map((d) => d.id)
-                : []
 
```

Comment:
I think you should call `updateDataview()` here, right?

Reply:
no, here we have the same problem